### PR TITLE
delete repo command and basicauth for creation

### PR DESCRIPTION
**Pull request by filip-halt**
Signed-off-by: Filip Haltmayer <filip.haltmayer@zilliz.com>

### CODE REVIEW

File: towhee/utils/hub_utils.py [hunk 1]
```diff
@@ -234,6 +255,29 @@ def create(self, password: str, repo_class: int) -> None:
...
+            r = requests.delete(url, headers={'Authorization': 'token ' + str(token['sha1'])})
+            r.raise_for_status()
+        except HTTPError as e:
+            if e.response.status_code == 404:
+                print('Repo not found.')
+            else:
+                raise e
+        finally:
+            self.delete_token(str(token['id']), password)
+
```
Comment (Chiiizzzy) [inline towhee/utils/hub_utils.py:280]: Junjie said we don't want to expose delete to users

File: towhee/utils/hub_utils.py [hunk 2]
```diff
@@ -197,6 +197,27 @@ def delete_token(self, token_id: int, password: str) -> None:
...
+            'trust_model': 'default',
+            'type': repo_class
+        }
+        url = self._root + '/api/v1/user/repos'
+        try:
+            r = requests.post(url, data=data, auth=HTTPBasicAuth(self._author, password))
+            r.raise_for_status()
+        except HTTPError as e:
+            raise e
+
```
Comment (Chiiizzzy) [inline towhee/utils/hub_utils.py:219]: Why do we need this?

Comment (shiyu22) [reply towhee/utils/hub_utils.py:219]: What is the difference between `create` and `basic_create` ? `basic_create` does not require a token?

Comment (filip-halt) [reply towhee/utils/hub_utils.py:219]: basic create doesnt use tokens, rather the login info. I believe there is an authorization error when you try to push to a repo that was made with tokens.

File: towhee/utils/hub_utils.py (hunk 1 above)
Comment (filip-halt) [reply towhee/utils/hub_utils.py:280]: Is there a specific reason for this @junjiejiangjjj

File: towhee/utils/hub_utils.py (hunk 2 above)
Comment (shiyu22) [reply towhee/utils/hub_utils.py:219]: Is that for #542

**Pull request closed without merging**